static_src/stores/service_instance_store.js

```diff
@@ -150,18 +161,27 @@ class ServiceInstanceStore extends BaseStore {
           action.spaceGuid,
           action.servicePlanGuid
         );
+        this._createLoading = true;
+        this.emitChange();
         break;
       }
 
       case serviceActionTypes.SERVICE_INSTANCE_CREATED: {
         cfApi.fetchServiceInstance(action.serviceInstance.guid);
-        this._createInstanceForm = null;
+        this._createLoading = false;
+        this._createdTempNotification = true;
```

Comment:
I think you want to clear the error on success here, just in case there was a previous failure we were displaying. Although not totally sure if it should be done on the CREATE action...